src/aiida/storage/psql_dos/orm/querybuilder/main.py

```diff
@@ -48,7 +48,6 @@
     },
     'db_dbnode': {
         'pk': 'id',
-        'dict': 'attributes',
         'computer_pk': 'dbcomputer_id',
         'user_pk': 'user_id',
```

Comment:
Why are you dropping this line? While we're discussing it, I think there might be an issue with `user_pk` as a field on a `Node` type. It doesn't seem to be a thing. `Node.user.pk` is, but not `user_pk`. Was this ever a field?